.github/workflows/release.yaml

```diff
@@ -0,0 +1,37 @@
+---
+name: Trigger release and bump version
+
+on:  # yamllint disable-line rule:truthy
+  pull_request:
+    types:
+      - closed
+
+jobs:
+  trigger-release-process:
+    if: github.event.pull_request.merged == true && startswith(github.event.head_commit.message, 'Release ')
```

Comment:
Can we make this condition more tighter somehow - I am a bit concerned with the condition for a new release being just a PR being merged with "Release" in the message

Reply:
I don't think we can easily do so through GitHub actions. Thus, I implement a simple python script for this validation.